<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrystalDialog</class>
 <widget class="QDialog" name="CrystalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create custom crystal structure</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddAtom">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Atom</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Preview</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>20</width>
     <height>461</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="selectCrystalSystem">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>76</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cubic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tetragonal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orthorhombic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hexagonal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monoclinic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Triclinic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(Trigonal)</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="selectSpaceGroup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>76</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>â€“</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m3m</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4/mmm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mmm</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatConst_a">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>110</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatConst_b">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>140</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatConst_c">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>170</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputAtomCoord_a">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>0.990000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputAtomCoord_b">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>0.990000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputAtomCoord_c">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>62</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>0.990000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputElementSymbol">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelElementSymbol">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Element</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoord_a">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>coord. a</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoord_b">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>coord. b</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoord_c">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>coord. c</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatConst_a">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatConst_b">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatConst_C">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitleLatConst">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattice constants</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputSupercell_a">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputSupercell_b">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="inputSupercell_c">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitleSupercell">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supercell</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSupercell_a">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;sub&gt;a&lt;/sub&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSupercell_b">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;sub&gt;b&lt;/sub&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSupercell_c">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;N&lt;sub&gt;c&lt;/sub&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="listAtoms">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>256</width>
     <height>291</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Atoms</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitleLatAngles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lattice angles</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatAngle_beta">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>270</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>179.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatAngle_alpha">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Î±</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatAngle_beta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Î²</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatAngle_alpha">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>240</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>179.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="inputLatAngle_gamma">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>300</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>179.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>90.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelLatAngle_gamma">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Î³</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CrystalDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CrystalDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
